src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -6,9 +6,76 @@
 
 from Utils.CherryPyPeriodicTask import CherryPyPeriodicTask
 from WMCore.Services.WMStats.WMStatsReader import WMStatsReader
+from WMCore.Services.WorkQueue.WorkQueue import WorkQueue
 from WMCore.Services.RequestDB.RequestDBWriter import RequestDBWriter
 from WMCore.Services.WMStats.DataStruct.RequestInfoCollection import RequestInfoCollection
 
+
+def moveForwardStatus(reqDBWriter, requestStatus):
+    
+    statusTransition = {"assigned": ["acquired", "running-open", "running-closed", "completed"], 
```

Comment:
Ok, I now see it's not the same allowed transitions mapping. Though it may be better to set this specific status transition in the same RequestStatus file.